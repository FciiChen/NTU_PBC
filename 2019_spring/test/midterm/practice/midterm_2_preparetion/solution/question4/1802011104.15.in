寧弈一直仰頭看著她，這個眼神令他心中一緊。

　　然而瞬間她便恢復如常，手一緊，更加有力的握住了他的手。

　　十指相扣，深深相攥，鳳知微的手和寧弈的腰同時用力，在絲繩上翻身而起，緊緊攀住絲繩。

　　翻身而起的時候，寧弈一腳踢在崖壁上，踢落一塊要掉不掉的浮石。

　　浮石轟隆隆的墜落下去，在極深極深的淵下發出空洞的回聲，聽起來就像一個人墜落。

　　吱呀一聲窗扇大開，二皇子探出頭來，有點遺憾的向下看，下方是茫茫雲海，什麼都看不清。

　　他皺眉望著崖下，低低道：「這傢伙，真稀鬆，我不過想先嚇嚇他問出主使再殺，他倒嚇得失足……可惜沒問出什麼來。」

　　他一點也沒抬頭看，縮回身子，砰一聲關上窗戶。

　　崖壁上緊緊相擁的兩人，同時撇嘴笑了笑。

　　鳳知微對寧弈做了個口型，問：「你沒事吧？」

　　寧弈不答，溫柔的看著她，一向沉冷的目光裡柔情微微綻放，蕩漾了星光，醉了月。

　　他突然伸出手指，小心而細緻的擦去她因為緊緊貼壁而下，而在臉上沾著的夜露和泥濘。

　　山壁嶙峋，摩擦得面具已經有了破口，好在沒傷著她的肌膚。

　　鳳知微有點不自然的躲著他的目光，指指崖上，示意翻上去，寧弈搖搖頭。

　　鳳知微一怔，隨即明白他的意思，黃大人既然被「滅口」，黃夫人必然也不能讓她活著出去，現在黃夫人休息的客房，已經很危險。

　　兩人對視一眼，鳳知微突然張口，在他耳邊輕輕道：「長寧使者。」

　　寧弈眼神裡露出讚許的笑意，點點頭，一揚下巴，指向崖的另一邊。

　　鳳知微知道他的意思是長寧藩的使者在那邊，那位使者不勝酒力，先前就去休息了，而且這人也沒有武功。

　　兩人此時貼在崖壁上，寧弈背靠山壁，單手扣著山壁，腳踩一塊突出的山石，鳳知微背貼著他的胸，被他緊擁在懷，山風吹得衣袂鼓蕩，烏髮交織飛舞，身下萬仞雲海，頭頂萬頃碧空，茫茫遠山奔入眼底，浩浩疆域極目馳騁，遠處蒼藍的天際裡，隱隱露出一線淡青色的晨曦，無邊無垠的伸展開去。

　　天地須彌，人如芥子。

　　極渺小，極廣大。

　　極危險，極曠朗。

　　雖然知道這不是沉迷的時機，兩人依舊為這一生裡難以再次遇見的浩闊場景而微微出神，風自幽邃盡頭奔來，滌蕩呼嘯，掃入心胸，兩人都似於同時聽見，彼此內心深處悠長的浩嘆。

　　江山多嬌，卻不與人共老。

　　半晌鳳知微輕輕一嘆，微濕的眼睫不知是露水還是別的什麼，簌簌眨落一點晶瑩。

　　寧弈抱緊了她，下巴在她柔滑的髮絲上蹭了蹭，眼神迷離。

　　鳳知微一笑，一抬手，手中匕首精光一亮，割斷了那截絲繩，隨即匕首在山壁上一戳，身子一翻翻上匕首，往長寧使者的房間爬去。

　　寧弈緊隨其後，護在她身下。

　　兩人很快爬到那間屋子的後窗下，漱玉山莊追求曠朗自然，使者的這間房間，後窗是大排的軒窗，雖然都關著，但也難不倒這兩人，鳳知微貼在崖壁上，用匕首去撬窗的插銷。

　　忽覺腿彎一暖，一低頭看見寧弈抱住了她的腿彎，看她看下來，也仰起臉，對她露出淡淡笑意。

　　今夜他的笑，和平日或蕩漾或風流裡都帶了幾分涼不同，難得的乾淨而溫暖，跳躍著微微的熱意，像永夜裡一點深紅的星火，遠，卻那般真實的暖著。

　　鳳知微怔在那樣的眼神裡，忽然想起那年，也是一處崖壁，不抵這高，不抵這冷，不抵這險，也有人輕輕抱住了另一人的腿彎，許諾要做另一人的眼睛。

　　那時往下，這時往上，那時是她抱住他，這時是他抱住她。

　　寧弈手掌的熱力透過來，似要深入肌骨，她顫了顫，一瞬間眼前掠過那年山寺的夜雨。

　　當年山寺聽夜雨，濕了誰袍角的落花。

　　鳳知微收回目光，垂下眼睫。

　　極輕微的「咔」一聲，窗銷被撥開。

　　無聲推開窗扇，肩頭一聳，寧弈抬手將她一送，鳳知微青煙般掠了進去。

　　一落地便直撲床榻，手中青光一閃已經遞了出去。

　　身後落足微響，寧弈也已經撲了進來。

　　鳳知微在一片凌厲的風聲裡，手掌控向那人咽喉，她身子輕盈輕功比寧弈還要高上一籌，這一全力撲出，瞬間便到那人身前。

　　對方沒有武功，卻被風聲驚動，惶然掀被而起。

　　黑暗中奇異的光芒一閃。

　　似乎是明亮又暗淡的淡銀光。

　　鳳知微眼角一瞥心中轟然一聲，暗叫不好，然而招式已經用老無法收回，百忙中只來得及一腳將一個凳子踢出去，擋住隨後撲來的寧弈。

　　銀光一閃，她伸出的手一痛，如同被鉗子鉗住，隨即身子一傾，已經被一股大力狠狠拽了過去。
BREAK