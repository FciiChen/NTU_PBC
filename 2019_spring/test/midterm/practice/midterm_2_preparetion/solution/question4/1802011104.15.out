鳳_微 15
不_這 4
在_壁 4
崖_上 4
了_去 3
抱_了 3
的_著 3
經_了 3
著_的 3
黃_人 3
